nautobot_data_validation_engine/forms.py

```diff
@@ -297,8 +299,8 @@ class DataComplianceFilterForm(BootstrapMixin, forms.Form):
     """Form for DataCompliance instances."""
 
     model = DataCompliance
-    compliance_class_name = forms.CharField(max_length=20, required=False)
-    validated_attribute = forms.CharField(max_length=20, required=False)
+    compliance_class_name = MultiValueCharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    validated_attribute = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
```

Comment:
Should this also be a `MultiValueCharField`?

Reply:
Yes